src/tests/JIT/HardwareIntrinsics/X86/X86Base/DivRem.RefOnly.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <!--
+    DivRem.CoreLib provides a "shim" definition of DivRem in System.Private.CoreLib
+    The DivRem test suite references this project at compile time but does not get deployed with the test.
+    As a result, the DivRem test suite has a direct reference to System.Private.CoreLib for the DivRem methods,
+    which enables us to avoid using Reflection.Emit to provide the implementations.
+  -->
+  <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <OutputType>Library</OutputType>
+    <CLRTestKind>SharedLibrary</CLRTestKind>
+    <AssemblyName>System.Private.CoreLib</AssemblyName>
```

Comment:
Referencing CoreLib from the tests is anti-pattern. Tests should only reference public surface. If the tests need to hack on internal implementation details, they should use private reflection. Why are we doing this?